pandas/tseries/tests/test_period.py

```diff
@@ -6,27 +6,25 @@
 
 """
 
+import numpy as np
+from numpy.random import randn
```

Comment:
sure can do in anthoher PR. I though you moved the PeriodIndex tests out already. But yes, certainly should split it.